### PR TITLE
Remove dynamic domain in sale_management

### PR DESCRIPTION
**Pull request by xmo-odoo**
Task 2115472

Not sure where I put the static domain?

### CODE REVIEW

File: addons/sale_management/models/sale_order.py [hunk 1]
```diff
@@ -202,8 +202,6 @@ def _onchange_product_id(self):
         if pricelist and product:
             partner_id = self.order_id.partner_id.id
             self.price_unit = pricelist.with_context(uom=self.uom_id.id).get_product_price(product, self.quantity, partner_id)
-        domain = {'uom_id': [('category_id', '=', self.product_id.uom_id.category_id.id)]}
-        return {'domain': domain}
```
Comment (Feyensv) [inline addons/sale_management/models/sale_order.py]: The static domain is already defined:
https://github.com/odoo/odoo/blob/master/addons/sale_management/models/sale_order.py#L176-L177

Comment (xmo-odoo) [reply addons/sale_management/models/sale_order.py]: Ah right, that's probably why, it's just that the dynamic domain was left over.

File: addons/sale_management/models/sale_order_template.py [hunk 2]
```diff
@@ -146,8 +146,6 @@ def _onchange_product_id(self):
             name += '\n' + self.product_id.description_sale
         self.name = name
         self.uom_id = product.uom_id
-        domain = {'uom_id': [('category_id', '=', self.product_id.uom_id.category_id.id)]}
-        return {'domain': domain}
```
Comment (Feyensv) [inline addons/sale_management/models/sale_order_template.py]: Same
https://github.com/odoo/odoo/blob/master/addons/sale_management/models/sale_order_template.py#L134-L135

**Comment by xmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**